packages/react-components/src/components/MessageStatusIndicator.tsx

```diff
@@ -127,6 +145,9 @@ export const MessageStatusIndicator = (props: MessageStatusIndicatorProps): JSX.
                   remoteParticipantsCount: `${remoteParticipantsCount}`
                 })
           }
+          onTooltipToggle={() => {
+            setIsToolTipHovered(!isToolTipHovered);
```

Comment:
This is onToolTipsToggle instead of tooltips hover, the behavior is hover on the element to toghle the tooltip, not hover on the tool tip